src/server/api/controllers/channels-controller.js

```diff
@@ -7,11 +7,7 @@ const createChannel = async (body) => {
   //  will return the total number of channels after inserting a new channel
   const insertedChannels = await knex('channels').insert(newChannel);
 
-  return {
-    successful: true,
-    //  id for the newly created channel will be same as total number of channels inserted
-    id: insertedChannels[0],
-  };
+  return { channelId: insertedChannels[0] };
```

Comment:
It is a convention to return `id` here rather than `channelId`. You can always rename in the frontend.
I'd suggest you don't change this on the server side.